org.gnome.IconPreview.json

```diff
@@ -14,16 +14,29 @@
         "--talk-name=ca.desrt.dconf",
         "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
     ],
+    "cleanup" : [
```

Comment:
I'd rather you didn't copy paste a random list. Only a few of these actually apply so please check your build directory and just do those.